Tasks/Gradle/task.loc.json

```diff
@@ -14,8 +14,8 @@
   ],
   "author": "Microsoft Corporation",
   "version": {
-    "Major": 1,
-    "Minor": 111,
+    "Major": 2,
```

Comment:
top level preview = true will achieve that.  If it is a large refactor or breaking changes, mark as preview.  If it's not, then it shouldn't be 2.x.  Pick one.